libs/api-sdk/src/apiSdk.ts

```diff
@@ -350,4 +351,31 @@ export default class ApiSdk {
 
         return invite
     }
+
+    /**
+     * Generate a custom url for joining a credential group.
+     * @param baseUrl Base url.
+     * @param groupId Group id.
+     * @param commitment Identity commitment.
+     * @param providerName Group credential provider name.
+     * @param redirectUri Redirect uri.
+     * @returns Url string.
+     */
+    joinCredentialGroup(
+        baseUrl: string,
```

Comment:
Since this will always be a dashboard url, we could have an enum similar to `SupportedUrl` with all the dashboard urls. What do you think?

You can find them all here: https://github.com/bandada-infra/bandada?tab=readme-ov-file#-configuration

Reply:
Got it, if it's only being used in dashboard then it makes sense to use enum for it. 

Do you foresee that the dashboard urls for `SupportedUrl` will be used in other places? If yes, then I'll add a new type `DashboardSupportedUrl`  in `types`.

Reply:
Yes, I think that the dashboard urls can be part of the types, similar to the `SupportedUrl`. It would be something like:

```ts
export enum DashboardUrl {
    DEV = "http://localhost:3001",
    PROD = "https://app.bandada.pse.dev",
    STAGING = "https://app-staging.bandada.pse.dev"
}
```

Reply:
Should I add in another one for `ClientUrl` as well?

Reply:
I don't think so because people could put the url they want there. They don't need to follow a specific structure. They just need to follow a structure for the dashboard url.

Reply:
Noted, pushed the requested changes :)